documentation/contributing/future_projects/index.markdown

```diff
@@ -56,6 +57,16 @@ Feel free to contact [PickNik Robotics](https://picknik.ai/connect/) for further
 
 - **Related Github issues**: [2287](https://github.com/ros-planning/moveit/issues/2287)
 
+## Grasp Pose Generation
+
+- **Prerequisites**: ROS
+- **Programming skills**: C++, Python
+- **Difficulty**: Medium
+- **Potential mentors**: Mark Moll
+- **Description**: Improve grasp pose synthesis within MoveIt and the MoveIt Task constructor. Current grasp synthesis algorithms pair deep neural networks and sampling point clouds, see [GPD](https://github.com/atenpas/gpd) and [Dex-Net](https://berkeleyautomation.github.io/dex-net/). An initial effort using the previous grasping methods includes a [demo](https://github.com/PickNikRobotics/deep_grasp_demo) and a [tutorial](https://ros-planning.github.io/moveit_tutorials/doc/moveit_deep_grasps/moveit_deep_grasps_tutorial.html). Further work is required to harden this implementation, see [PR 196](https://github.com/ros-planning/moveit_task_constructor/pull/196).
```

Comment:
I guess this says you won't have the time anymore to address this yourself?

Reply:
I am working right now to finish the changes Robert requested and hopefully will get that PR merged within a couple weeks.